gtsfm/utils/graph.py

```diff
@@ -53,6 +55,8 @@ def prune_to_largest_connected_component(
         Subset of unit_translations which are in the largest connected components.
     """
     input_edges = [k for (k, v) in rotations.items() if v is not None]
+    # TODO: start using pose priors by uncommenting the following line
+    # input_edges += [k for (k, v) in pose_priors.items() if v is not None]
```

Comment:
are you merging before this TODO is done?

Reply:
Yes. Any logic related change is out-of-scope here. I just want to get all the required inputs to the classes for now.